test/unit/test-extension-location.js

```diff
@@ -203,3 +204,192 @@ describes.sandboxed('Extension Location', {}, () => {
     });
   });
 });
+
+describes.sandboxed('Module Extension Location', {}, () => {
```

Comment:
Nit: I may be missing something, but this number of tests feels like overkill to me.
I think we'd still get good coverage with just one or two tests within the `describe('get correct script source')` grouping.

Reply:
agree with you for sure, was just in a rush and duplicated the tests